content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md

```diff
@@ -87,7 +87,11 @@ The project containing the sample CorDapp opens.
 This command builds the CorDapp package (`.cpk`) files. Most CorDapps have two files: a `contracts` file and a `workflows` file. The command also builds the CorDapp package bundle (`.cpb`).
 
 4. Deploy the network using `corda-cli` and `docker-compose`:
-  `corda-cli network deploy -n solar-system | docker-compose -f - up`
+```
+corda-cli network deploy -n solar-system -f solar-system.yaml | docker-compose -f - up -d
+```
+
+Here the `-f` flag ensures that the network files have the same name as the network itself.
```

Comment:
We could omit the -f from the previous command as the network and network definition yaml have the same name, not sure if we need to mention it though, it should more be something documented on the command guide page where we explain what all the flags do.